app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -682,7 +682,17 @@ class ComposeActivity : BaseActivity(),
     }
 
     private fun updateVisibleCharactersLeft() {
-        composeCharactersLeftView.text = String.format(Locale.getDefault(), "%d", maximumTootCharacters - calculateTextLength())
+        var remainingLength = maximumTootCharacters - calculateTextLength();
+        composeCharactersLeftView.text = String.format(Locale.getDefault(), "%d", remainingLength)
+
+        if(remainingLength < 0) {
+            val textColor = ThemeUtils.getColor(this, android.R.attr.colorError)
+            composeCharactersLeftView.setTextColor(textColor)
+        }
+        else {
+            val textColor = ThemeUtils.getColor(this, android.R.attr.textColorTertiary)
+            composeCharactersLeftView.setTextColor(textColor)
+        }
```

Comment:
You can write this a bit shorter:
```suggestion
        val textColor = if(remainingLength < 0) {
            ThemeUtils.getColor(this, android.R.attr.colorError)
        } else {
            ThemeUtils.getColor(this, android.R.attr.textColorTertiary)
        }
        composeCharactersLeftView.setTextColor(textColor)
```